logstash-core/spec/logstash/event_spec.rb

```diff
@@ -319,10 +319,8 @@
     end
 
     it "should consistently handle nil" do
-      blank_strings.each do |s|
-        expect{LogStash::Event.from_json(nil)}.to raise_error
-        expect{LogStash::Event.new(LogStash::Json.load(nil))}.to raise_error
-      end
+      expect{LogStash::Event.from_json(nil)}.to raise_error # TypeError
```

Comment:
instead of just commenting the error type, let's assert it. If we don't like what is being raised in practice, we should take this as an opportunity to refine the implementation. BWC-wise, we only promise to raise here so raising something more specific and specifying that here is not a breaking change.